lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/Transactions.hs

```diff
@@ -170,33 +173,24 @@ spec = do
              ]
 
     it "TRANS_EXTERNAL_CREATE_01 - proper single output transaction and \
-       \proper binary format" $ \ctx -> do
-
-        let toSend = 1 :: Natural
-        (tx, wits) <- fixtureExternalTx ctx toSend
-        let encodedSignedTx = encodeTx (tx, wits) MsgTypeTransaction
-        let payload = Json [json|{
-                "payload": #{encodedSignedTx}
-            }|]
-        r <- request @ApiTxId ctx postExternalTxEp Default payload
-        verify r
-            [ expectSuccess
-            , expectResponseCode HTTP.status202
-            ]
+       \proper binary format - octet-stream" $ \ctx -> do
+        postExternalTxTest ctx
 
     it "TRANS_EXTERNAL_CREATE_02 - proper single output transaction and \
        \wrong binary format" $ \ctx -> do
-
         let toSend = 1 :: Natural
-        (tx, wits) <- fixtureExternalTx ctx toSend
-        let wronglyEncodedSignedTx = encodeTx (tx, wits) MsgTypeInitial
-        let payload = Json [json|{
-                "payload": #{wronglyEncodedSignedTx}
-            }|]
-        r <- request @ApiTxId ctx postExternalTxEp Default payload
+        (ExternalTxFixture _ _ _ txWits) <-
+                fixtureExternalTx ctx toSend
+        let baseOk = Base16
+        let wronglyEncodedSignedTx = encodeTx txWits MsgTypeInitial baseOk
+        let payload =
+                NonJson $ BL.fromStrict $
+                (toRawBytes baseOk) wronglyEncodedSignedTx
```

Comment:
For this one, perhaps it's possible to save a line?
```hs
        let payload = NonJson $ BL.fromStrict $
                (toRawBytes baseOk) wronglyEncodedSignedTx
```

Reply:
done